libraries/ec2.rb

```diff
@@ -69,8 +69,13 @@ def fallback_region
 
     # setup AWS instance using passed creds, iam profile, or assumed role
     def create_aws_interface(aws_interface, **opts)
+      use_fips_endpoint = false
+      if opts.key?(:use_fips_endpoint)
+        use_fips_endpoint = opts[:use_fips_endpoint]
+      end
```

Comment:
Simplified into an inline if statement
```suggestion
      use_fips_endpoint = opts.key?(:use_fips_endpoint) ? opts[:use_fips_endpoint] : false
```